app/(main)/agents-v2/page.tsx

```diff
@@ -23,20 +15,12 @@ function DataList({ label, children }: { label: string; children: ReactNode }) {
 }
 
 async function AgentList() {
-	const user = await getUser();
+	const currentTeam = await fetchCurrentTeam();
 	const dbAgents = await db
 		.select({ id: agents.id, name: agents.name, updatedAt: agents.updatedAt })
 		.from(agents)
-		.innerJoin(teamMemberships, eq(agents.teamDbId, teamMemberships.teamDbId))
-		.innerJoin(
-			supabaseUserMappings,
-			eq(teamMemberships.userDbId, supabaseUserMappings.userDbId),
-		)
 		.where(
-			and(
-				eq(supabaseUserMappings.supabaseUserId, user.id),
-				isNotNull(agents.graphUrl),
-			),
+			and(eq(agents.teamDbId, currentTeam.dbId), isNotNull(agents.graphUrl)),
```

Comment:
Am I right in understanding that the agents of other users belonging to the same team are also displayed?
Also, is it possible to determine that they were created by you?

Reply:
@toyamarinyon 

> Am I right in understanding that the agents of other users belonging to the same team are also displayed?

Yes, right.

> Also, is it possible to determine that they were created by you?

No.
For this purpose, we must add a `teams.user_db_id` column. (or `teams.creator_db_id`)

Reply:
@satococoa We want to retain the creator. There are possible use cases to display or refine the creator.

@shige Can we have @satococoa proceed with the support to add columns?

Reply:
@toyamarinyon @satococoa 

For now, since the code changes in this PR are extensive, I’d like to proceed with them in a separate PR.

My Thoughts:  
- I’d like to incorporate the concept of an agent creator, such as `agent.creator`, `agent.creator_db_id`.  
- It might be used later for displaying the list of agents, for example.  
- Before the public release of the team plan, I want to ensure the data is saved in the database table.

Reply:
@toyamarinyon @shige 

Thank you!
I'll create another pull request to add `agent.creator_db_id` (with migration script), next.